client/src/components/common/Forms/BasicForm/Forms/CustoForms/ResistrationForm.tsx

```diff
@@ -0,0 +1,47 @@
+import InputItem from "../../../../Input/InputItem";
+import BasicSelect from "../../../../../Select/Select";
+import { selectRoleColumns } from "../../../../../../config/config";
+
+const RegistrationForm = ({ onChange, columns, userId, submitForm }: any) => {
+  const handleFormSubmit = (e: any) => {
+    e.preventDefault();
+    submitForm();
+  };
+
+
+  return (
+    <main className="form-signin text-center d-flex justify-content-center col-12">
+      <form className="col-6">
+        <h1 className="h3 mb-3 fw-normal">Please sign up</h1>
+        {columns.map((column: any) => {
+          return (
+            <InputItem
+              onChangeEvent={onChange}
+              type={column.type}
+              className={column.className}
+              id={column.id}
+              placeholder={column.placeholder}
+              label={column.label}
+              fieldName={column.fieldName}
+            />
+          );
+        })}
+
+        <BasicSelect
+          onChangeEvent={onChange}
+          columns={selectRoleColumns}
+          title="Select person role"
+        />
+        <button
```

Comment:
you use bootstrap and material-ui, don't change it,

but note bootstrap and material-ui have mostly duplicated functionality, so better to use only one UI kit for a project